extensions/axon/runtime/src/main/java/io/quarkus/axon/runtime/AxonRuntimeConfig.java

```diff
@@ -0,0 +1,86 @@
+package io.quarkus.axon.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * The Axon configuration.
+ */
+@ConfigRoot(name = "axon", phase = ConfigPhase.RUN_TIME)
+public class AxonRuntimeConfig {
```

Comment:
Generally we try to limit the number of runtime configuration properties to the minimum - meaning we limit to things that make sense for the user to change at runtime (this is different from other frameworks which most of use are used to).

Do all of these properties need to be changeable at runtime?